src/components/pegout/PegOutForm.vue

```diff
@@ -62,8 +69,25 @@ import RskFeeSelect from '@/components/pegout/RskFeeSelect.vue';
 export default class PegOutForm extends Vue {
   environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
 
-  recipientAddress = '0xtesttesttesttesttest';
+  recipientAddress = '';
+
+  showAddressDialog = false;
 
   focus = false;
+
+  @Emit()
+  closeAddressDialog() {
+    this.showAddressDialog = false;
+  }
+
+  signTransaction() {
+    this.recipientAddress = '0xtesttesttesttesttest';
+    this.showAddressDialog = false;
+  }
```

Comment:
Change the name to `signMessage`,  The transaction will be signed in another function.